tools/nnyeah/integration/API/IntegrationAPI.cs

```diff
@@ -53,6 +68,7 @@ public NIntAPI ()
 	public nint ToNint (char a) => a;
 	public nint ToNint (int a) => a;
 	public nint PlusOne (nint a) => a++;
+	public nint NumberZero () => new MyNum ().Zero ().NIntValue;
```

Comment:
something funny in the indentation here, right? Happens with all the methods from line 55 'til this one.